test/controllers/api/deploys_controller_test.rb

```diff
@@ -3,7 +3,11 @@
 SingleCov.covered!
 
 describe Api::DeploysController do
+  assert_route verb: "GET", path: "/api/deploys/active_count", to: 'api/deploys#active_count'
```

Comment:
not sure if these are worthwhile, the test that executes them makes sure they exist ...